docs/entity_summary.md

```diff
@@ -4,11 +4,13 @@ The entity summary is the entity's main page, and contains all the information a
 
 * Dashboards should be listed in the `definition.yml` file, under the `dashboardTemplates` section.
 * If the entity type only has one source of data, use `newRelic` as the default source.
+* By default, each NRQL query will be filtered by the entity GUID. If you want to disable this behavior, set `filterNRQLByEntityID` to `false`. When you disable this behavior, you must manually specify the filter for all NRQL queries in the dashboard. See [Filtering by Entity GUID](#filtering-by-entity-guid) for more details.
```

Comment:
How are we going to make sure that the filter is passed into the query when the `filterNRQLByEntityID` is `false`?

Reply:
I understand, its not a problem.